source/__tests__/new/source-profile.test.js

```diff
@@ -0,0 +1,18 @@
+describe('Sinopia Profile Source Field', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://localhost:8000/#/profile/create/')
+  })
+
+  it('finds in the profile', async () => {
```

Comment:
I like that these tests have been written for the source field, but I would include them in the new-profile.test.  Maybe

```
describe('Source url'), async () => {
  it('source input field')
  it('popover text')
  it('validates with checkURL') // ensure the attrib is present on the element?
}
```

I would also consider adding methods such as 

```
  async function expect_value_in_selector_textContent(sel, value) {
    const sel_text = await page.$eval(sel, e => e.textContent)
    expect(sel_text).toBe(value)
  }

  async function expect_sel_to_exist(sel) {
    const sel_text = !!(await page.$(sel))
    expect(sel_text).toEqual(true)
  }
```

to that file to dry it up.

I would also try to find a better selector, if possible.  What do you think of `#profile input[name="source"]` for the input field (check that it exists) and `#profile label[for="source"]` with textContent Source for the label?